src/components/shared/TreeView.css

```diff
@@ -11,9 +11,20 @@
   user-select: none;
 }
 
+.treeViewRow,
 .treeViewHeader {
-  position: relative;
-  height: 16px;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: flex-start;
+  white-space: nowrap;
+}
+
+.treeViewRowScrolledColumns {
+  align-items: center;
+}
+
+.treeViewHeader {
+  height: 17px;
```

Comment:
It was 16px before, but this wasn't right. Indeed the line-height is 15px + 1px top and bottom, makes 17px.